extensions/positron-python/package.nls.json

```diff
@@ -151,7 +148,7 @@
     "walkthrough.step.python.createEnvironment.title": "Select or create a Python environment",
     "walkthrough.step.python.createEnvironment.description": {
         "message": "Create an environment for your Python project or use [Select Python Interpreter](command:python.setInterpreter) to select an existing one.\n[Create Environment](command:python.createEnvironment)\n**Tip**: Run the ``Python: Create Environment`` command in the [Command Palette](command:workbench.action.showCommands).",
-        "comment":  [
+        "comment": [
```

Comment:
There are some funky formatting inconsistencies between `positron` and the `positron-python` (that hopefully should be resolved soon with #5838) that can result in this weirdness via format-on-save 😩 

Are you able to remove these unrelated whitespace changes? I believe the easiest way is to command palette `Format Document` from inside the `extensions/positron-python` workspace.

Reply:
Sure! I originally kept them because they seemed like improvements, but I can remove them from this PR since they're unrelated.

Reply:
Oh I see, we inherited the weirdness from the upstream. :)